src/common/components/cards/urls-card-row.tsx

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { NewTabLink } from 'common/components/new-tab-link';
+import * as React from 'react';
+
+import { CardRowProps } from 'common/configs/unified-result-property-configurations';
+import { NamedFC } from 'common/react/named-fc';
+import { SimpleCardRow } from './simple-card-row';
+
+export interface UrlsCardRowProps extends CardRowProps {
+    propertyData: string[];
+}
+
+export const UrlsCardRow = NamedFC<UrlsCardRowProps>('UrlsCardRow', ({ deps, ...props }) => {
+    const renderUrlContent = () => {
+        return (
+            <div>
+                {props.propertyData.map((url, index) => (
+                    <div key={`urls-${index}`}>
```

Comment:
I wonder if it would be better to use a `<ul>` of URLs; even if we aren't styling it with bullet points, it feels semantically like a list to me. @ferBonnin, thoughts?

Reply:
I like that suggestion, it would make AT navigation and overall readability easier.

Reply:
A <ul> works for me, I didn't use one because bullet points were not part of the design. To clarify @ferBonnin, do you still want this visually the same (no bullets)?

Reply:
lets add visuall bullet points as well, please; I believe it would help for readability

Reply:
Now a bulleted `<ul>`! See screenshot.